conftest.py

```diff
@@ -23,7 +23,10 @@ def pytest_configure(config):
     module_name = config.getoption("controller")
 
     if module_name is None:
-        pytest.exit("--controller is required.", 1)
+        print("Missing --controller option, errors may occur.")
```

Comment:
What currently happens if you run without --controller or --help? Any way to get it to stop cleanly in that case?

Anyway, not a huge deal, feel free to merge.

Reply:
All tests except the self-test crash like this:

```
    def setUp(self) -> None:
        super().setUp()
        self.server_support = None
        self.find_hostname_and_port()
>       self.controller.run(
            self.hostname,
            self.port,
            password=self.password,
            valid_metadata_keys=self.valid_metadata_keys,
            invalid_metadata_keys=self.invalid_metadata_keys,
            ssl=self.ssl,
        )
E       AttributeError: 'ZncPlaybackTestCase' object has no attribute 'controller'

irctest/cases.py:467: AttributeError
```